app/helpers/application_helper.rb

```diff
@@ -1,4 +1,12 @@
+require "pagy/extras/shared"
```

Comment:
Any reason this is here and not in the initialiser?

Reply:
i will move, this was a leftover of me trying to get pagy to work

Reply:
turns out it's not needed any more so i've removed